docusaurus/docs/manage/02-router/03-sizing.md

```diff
@@ -0,0 +1,229 @@
+---
+id: router-sizing
+title: Sizing
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import styles from './styles.module.css';
+
+<Tabs>
+<TabItem value="vcpe" label="vCPE (generic type)" attributes={{className: styles.custom}}>
```

Comment:
I feel like "vCPE" is a meaningless term in this context, especially if it has to be explained as "generic". How about simply "Generic"?

Reply:
sure, sounds like a good idea